pkgs/development/python-modules/gffutils/default.nix

```diff
@@ -0,0 +1,29 @@
+{ python, stdenv, buildPythonPackage, fetchPypi
+, pyfaidx, six, argh, argcomplete, simplejson, nose, wget }:
+buildPythonPackage rec {
+  version = "0.10.1";
+  pname = "gffutils";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "a8fc39006d7aa353147238160640e2210b168f7849cb99896be3fc9441e351cb";
+  };
+
+  checkInputs = [ nose wget ];
+  propagatedBuildInputs = [ pyfaidx six argh argcomplete simplejson ];
+
+  checkPhase = ''
+    # sh gffutils/test/data/download-large-annotation-files.sh
+    # nosetests
```

Comment:
why no test suite?
```suggestion
    # nosetests
```

Reply:
The tests fail inside the nix environment

Reply:
general consensus is to just ignore test cases that fail due to sandbox environment. But some test failures may be indications that the package is in an actual broken state.

tests are also very useful for verifying that dependency bumps don't cause regressions.